src/base/solvers/configEnvVars.m

```diff
@@ -186,7 +186,8 @@ function configEnvVars(printLevel)
 
         % check for 32-bit
         if ~isempty(strfind(computer('arch'), '64'))
-            subDir = [filesep, osPath, filesep, 'matlab'];
+            subDir = [filesep, 'matlab'];
+            %subDir = [filesep, osPath, filesep, 'matlab'];
         end
```

Comment:
We should not be maintaining the installation instructions for gurobi or any other solver. We should refer the reader to the solver installation instructions.
http://www.gurobi.com/documentation/7.0/quickstart_linux/software_installation_guid.html#section:Installation

In gurobi_setup() ..
The cobra toolbox should look for $GUROBI_HOME, on all platforms
The installation instructions ask for...
export GUROBI_HOME="/opt/gurobi702/linux64" 

This is what my one is
$ echo $GUROBI_HOME
/usr/local/bin/gurobi702/linux64

It took a long time to debug that problem and was made more difficult by the lack of commenting, e.g, in configEnvVars.m

K>> subDir = [filesep, 'matlab']
subDir =
/matlab
K>> [solverPaths{k, 1}, ' = [', solverPaths{k, 1}, ', ''', subDir, '''];']
ans =
GUROBI_PATH = [GUROBI_PATH, '/matlab'];
K>> GUROBI_PATH
GUROBI_PATH =
/usr/local/bin/gurobi702/linux64
K>> [solverPaths{k, 1}, ' = [', solverPaths{k, 1}, ', ''', subDir, '''];']
ans =
GUROBI_PATH = [GUROBI_PATH, '/matlab'];
K>> eval([solverPaths{k, 1}, ' = [', solverPaths{k, 1}, ', ''', subDir, '''];'])
K>> GUROBI_PATH
GUROBI_PATH =
/usr/local/bin/gurobi702/linux64/matlab
K>> ls('/usr/local/bin/gurobi702/linux64/matlab')
gurobi_iis.m	   gurobi.m	  gurobi_read.m       gurobi_setup.m  gurobi_write.mexa64
gurobi_iis.mexa64  gurobi.mexa64  gurobi_read.mexa64  gurobi_write.m  README

This seems correct. You may not notice it when you are not reinstalling solvers from scratch.

Lines 13-64 in configEnvVars.m should be the backup, not the first line approach.